pom.xml

```diff
@@ -1135,6 +1136,12 @@
         <version>${spark.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.dropwizard.metrics</groupId>
+        <artifactId>metrics-core</artifactId>
```

Comment:
Generally speaking, when taking a new category of dependencies, we should do a full analysis to understand the dependency and its transitive dependencies, including license, API surface, size, bloat, etc. of all of dependencies brought in directly or transitively.